build/root/usr/local/bin/ci_entrypoint_nfd-operator.sh

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env bash
+
+set -o pipefail
+set -o errexit
+set -o nounset
+
+prepare_cluster_for_nfd() {
+    toolbox/cluster/capture_environment.sh
+}
```

Comment:
I guess you're missing the missing `!` :)
in plain english: 
```
# *before* installing and testing NFD
if NFD labels are found in the cluster nodes, then
  exit, because NFD must be already installed (this is unexpected), so we cannot test our master deployment
fi
```

Reply:
labels and NFD running are 2 diff things, if so , the undeploy playbook should also remove the labels, and is not doing that currently

Reply:
I will add a labels clean to undeploy to do this